styles/colors.css

```diff
@@ -0,0 +1,126 @@
+/* Colors of crystal ui styles below */
+.ct-colors {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 1rem;
+  background-color: #fafafa;
```

Comment:
Color variable  can be used

Reply:
I have put explicitly coz dark mode was giving problem